### DISCUSSION TITLE
No Downloads Folder Access in Security and Privacy.

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#4947] [UTM Help (macOS)]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
No Disk access... it looks like full disk access is needed to load CD-ROM images and there is no prompt in Privacy and Security.

**Configuration**  
* UTM Version:  4.15
* macOS Version: 12.6.2
* Mac Chip (Intel, M1, ...): M1

**Crash log**  
No Crash Log.... I just couldn't open any CD-ROMS

**Debug log**  
No debit messages....

**Upload VM**  
No VM... I tried with several versions... OpenBSD, Ubuntu and FreeBSD. 
<img width="1440" alt="Screen Shot 2023-01-07 at 4 53 40 PM" src="https://user-images.githubusercontent.com/102542366/212569689-6dead4e5-4b45-461c-b8ad-a60657f59256.png">

<img width="1440" alt="Screen Shot 2023-01-07 at 4 53 40 PM" src="https://user-images.githubusercontent.com/102542366/212569713-0325f504-f646-4e12-9a97-1f0081660213.png">

**Comment by osy** [comment]
When you mount an ISO it will grant permissions to that file.
